ttnn/cpp/ttnn/operations/full_like/device/full_like_factory.cpp

```diff
@@ -0,0 +1,132 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+
+#include "common/constants.hpp"
+#include "full_like_device_operation.hpp"
+#include "host_api.hpp"
+#include "impl/buffers/circular_buffer_types.hpp"
+#include "tt_metal/common/work_split.hpp"
+#include "ttnn/tensor/types.hpp"
+
+namespace ttnn::operations::full_like {
+
+using namespace tt;
+using namespace tt::tt_metal;
+using namespace tt::constants;
+
+union datatype {
+    uint32_t u32;
+    float f32;
+} u;
+
+FullLikeOperation::ProgramFactory::cached_program_t FullLikeOperation::ProgramFactory::create(
+    const operation_attributes_t& operation_attributes,
+    const tensor_args_t& tensor_args,
+    tensor_return_value_t& output) {
+    auto input = tensor_args.input;
+    auto fill_value = operation_attributes.fill_value;
+    if (std::holds_alternative<int>(fill_value)) {
+        u.u32 = std::get<int>(fill_value);
+    } else if (std::holds_alternative<float>(fill_value)) {
+        u.f32 = std::get<float>(fill_value);
+    }
+    DataType dtype{operation_attributes.dtype};
+    Layout layout{operation_attributes.layout};
+    Device* device = input.device();
+    MemoryConfig memory_config{operation_attributes.memory_config};
+
+    auto num_tiles = input.volume() / TILE_HW;
+
+    Program program{};
+
+    tt::DataFormat data_format = tt::tt_metal::datatype_to_dataformat_converter(dtype);
+    uint32_t single_tile_size = tt::tt_metal::detail::TileSize(data_format);
```

Comment:
```suggestion
    using namespace tt;
    using namespace tt::tt_metal;
    using namespace tt::tt_metal::detail;
    
    auto data_format = datatype_to_dataformat_converter(dtype);
    uint32_t single_tile_size = TileSize(data_format);
```
Please use `using namespace`. The TT example is also using it.

Reply:
Resolved